src/js/components/award/shared/AwardPageWrapper.jsx

```diff
@@ -32,7 +31,7 @@ const AwardPageWrapper = ({
         if (!areDefCodesLoading) {
             setCovidDefCodes(defCodes.filter((c) => c.disaster === 'covid_19' && allDefCodes.indexOf(c.code) > -1).map((code) => code.code));
         }
-    }, [areDefCodesLoading, allDefCodes]);
+    }, [areDefCodesLoading, allDefCodes, defCodes]);
```

Comment:
was this addition intentional?

Reply:
no, it was auto-added on save but i left it bc i don't think it's causing problems